ejb3/src/main/java/org/jboss/as/ejb3/subsystem/ApplicationSecurityDomainService.java

```diff
@@ -135,11 +135,7 @@ public boolean isEnableJacc() {
             return enableJacc;
         }
 
-        public BiFunction<String, ClassLoader, Registration> getSecurityFunction() {
-            return this::registerElytronDeployment;
-        }
-
-        private Registration registerElytronDeployment(final String deploymentName, final ClassLoader classLoader) {
+        public Registration apply(final String deploymentName, final ClassLoader classLoader) {
```

Comment:
This class is the API exposed by a public capability[1]. I think it's likely safe enough to change the contract like this but should there be some sort of javadoc about what this method does?

[1] There's no wildfly-capabilities entry but it can't be private as a foreign subsystem, webservices, uses it.

Reply:
@darranl please add the javadoc